sdk/go/common/resource/plugin/plugin.go

```diff
@@ -296,7 +296,7 @@ func newPlugin(ctx *Context, pwd, bin, prefix string, kind apitype.PluginKind,
 		killerr := plug.Kill()
 		contract.IgnoreError(killerr) // ignoring the error because the existing one trumps it.
 		return nil, fmt.Errorf(
-			"%v plugin [%v] wrote a non-numeric port to stdout ('%v'): %w", prefix, bin, port, err)
+			"%v plugin [%v] wrote a non-numeric port to stdout: %w", prefix, bin, err)
```

Comment:
Does `err` contain the port here?  Otherwise we could just print `portString` here instead to get the information we actually wanted?

Reply:
Yes, it looks like this at the moment for example:
```
error: testprovider (resource) plugin [/home/runner/work/pulumi-dotnet/pulumi-dotnet/integration_tests/testprovider/pulumi-resource-testprovider] wrote a non-numeric port to stdout ('0'): strconv.Atoi: parsing "\x1b]9;4;3;\x1b\\\x1b]9;4;0;\x1b\\ 43387": invalid syntax 
```